int32 PRE_POSE=0
int32 DEMO_TRAJ=1
int32 SHY=2
int32 HUG=3
int32 SHAKE=4
int32 STICK_EM_UP=5
int32 WAVE=6
#
int32 playfile_code
---
int32 DID_NOT_FIND_PLAYFILE=0
int32 FOUND_RIGHT_ARM_PLAYFILE=1
int32 FOUND_LEFT_ARM_PLAYFILE=2
int32 FOUND_BOTH_ARMS_PLAYFILES=3
int32 UNKNOWN_CASE=4
int32 return_code
